packages/convenient_test_manager/test/main_test.dart

```diff
@@ -1,5 +1,44 @@
+import 'dart:io';
+
+import 'package:convenient_test_common_dart/convenient_test_common_dart.dart';
+import 'package:convenient_test_manager/main.dart';
+import 'package:convenient_test_manager/misc/setup.dart';
+import 'package:convenient_test_manager/services/misc_flutter_service.dart';
+import 'package:convenient_test_manager_dart/services/vm_service_wrapper_service.dart';
+import 'package:flutter/material.dart';
 import 'package:flutter_test/flutter_test.dart';
 
+import 'fake_vm_service_wrapper.dart';
+
+var isSetup = false;
 void main() {
-  test('dummy', () {});
+  // color scheme breaks goldens, so we need two variants
+  myAppGoldenTest(ThemeMode.light);
+  myAppGoldenTest(ThemeMode.dark);
+}
+
+Future<void> myAppGoldenTest(ThemeMode theme) async {
+  testWidgets('UI Golden theme=$theme', (tester) async {
+    await tester.binding.setSurfaceSize(const Size(1920, 1080));
+
+    if (!isSetup) {
+      await setup(registerVmServiceWrapper: false, initVLC: false, parseConfigFile: false, setWinSize: false);
+
+      getIt.registerSingleton<VmServiceWrapperService>(FakeVmServiceWrapper());
+      isSetup = true;
+      Log.d('myAppGoldenTest', 'setup finished');
+    }
```

Comment:
Would be great to avoid this "setup once" logic.

For example, we may do this:

```
void main() {
      getIt.registerSingleton<VmServiceWrapperService>(FakeVmServiceWrapper());
  // NOTE
  setUpAll(() async => await setup());

  testWidgets(...);
}
```

By doing so, we are sure the setup is run once, while avoid this global variable logic.